hdfs-commons/src/main/java/org/apache/mesos/hdfs/config/HdfsFrameworkConfig.java

```diff
@@ -67,21 +70,26 @@ public HdfsFrameworkConfig(Configuration conf) {
   }
 
   private Configuration getEnvConfiguration() {
-    Configuration cfg = new Configuration(false);
-
-    for (Map.Entry<Object, Object> property : System.getProperties().entrySet()) {
-      String name = property.getKey().toString();
-      if (!name.startsWith("MESOS_")) {
-        continue;
-      }
```

Comment:
We are using "MESOS" multiple times. Let's declare it as a string (or add it to the constants if we think it will be used elsewhere)